src/api/users.ts

```diff
@@ -0,0 +1,37 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const usersRouter = express.Router();
+
+// GET /api/users
+
+usersRouter.get("/", async (req, res, next): Promise<void> => {
+    try {
+        const users = await prisma.user.findMany();
```

Comment:
Also add a TODO to strip the passwords from these users - it would be a security issue if you forget to do that later